sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -98,6 +98,26 @@
     background-color="error-bg"
     content="You can review the outcomes of your requested changes in the table below by clicking “View request”. Please note your application will proceed without your requested changes, based on your last assessment."
   />
+  <application-status-tracker-banner
+    label="Attention! You are not yet eligible to receive funding."
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error-bg"
+    content="You have applied for disability funding on your application, but your
+      disability status on your student profile has not yet been verified. Only
+      once your status is verified will you be able to receive funding."
```

Comment:
Is there a reason for this hardcoded message to be here?
I was expecting only the dynamically generated ones.

Reply:
I'll remove it.